redmine/manifests/redmine-svc.yaml

```diff
@@ -9,8 +9,9 @@ metadata:
 spec:
   type: LoadBalancer
   ports:
-  - port: 80
-    targetPort: redmine-http
+  - name: http
```

Comment:
should we name the port `www` instead of `http`? IANA defines port `80` as `www` service (http://www.speedguide.net/port.php?port=80).